### ISSUE TITLE
Python 3.12 SyntaxWarning: invalid escape sequence '\/'

### ISSUE THREAD

**Issue by kaistian** [mk-fg/python-pulse-control#82]
I'm packaging 24.4.0 for Void Linux, but under install in the byte-compiling step I get this warning.
```
Byte-compiling python3.12 code for module pulsectl...
usr/lib/python3.12/site-packages/pulsectl/lookup.py:24: SyntaxWarning: invalid escape sequence '\/'
  '''Return set of pulse object(s) with proplist values matching lookup-string.
usr/lib/python3.12/site-packages/pulsectl/lookup.py:24: SyntaxWarning: invalid escape sequence '\/'
  '''Return set of pulse object(s) with proplist values matching lookup-string.
```
This warning is discussed in https://github.com/python/cpython/issues/120113

Is this something that could be fixed for the next version please?

**Comment by mk-fg** [comment]
Should be fixed by e22a05b in 24.8.0. Thanks for reporting.

That bundled module is more of an undocumented proof-of-concept for an earlier issue, so unlikely to be imported or used by anyone, and tests don't import it either, so warning is probably only visible in something like packaging tools that compile all .py files.

**Comment by kaistian** [comment]
Thanks for the quick fix.
It's the client that does the compiling so every user installing the package would have seen the warning so I appreciate the fix.
